test/mpi/pt2pt/testlist.def

```diff
@@ -48,3 +48,10 @@ dtype_send 2
 recv_any 2
 irecv_any 2
 large_tag 2
+hbm 2 arg=-membind=FASTBOXES:HBM,CELLS:DRAM,COPYBUFS:DRAM
+hbm 2 arg=-membind=FASTBOXES:DRAM,CELLS:HBM,COPYBUFS:DRAM
+hbm 2 arg=-membind=FASTBOXES:DRAM,CELLS:DRAM,COPYBUFS:HBM
+hbm 2 arg=-membind=FASTBOXES:HBM,CELLS:HBM,COPYBUFS:DRAM
+hbm 2 arg=-membind=FASTBOXES:HBM,CELLS:DRAM,COPYBUFS:HBM
+hbm 2 arg=-membind=FASTBOXES:HBM,CELLS:HBM,COPYBUFS:HBM
+hbm 2 arg=-membind=FASTBOXES:DRAM,CELLS:HBM,COPYBUFS:HBM
```

Comment:
Do we need to specify `DRAM`?  Isn't that the default?  Or are you simply doing it for completeness?

Reply:
Yes just for completeness